omod/src/main/resources/config.xml

```diff
@@ -58,6 +58,11 @@
 			<path>/lib/coreapps-api-2.2-${project.parent.version}.jar</path>
 			<openmrsVersion>2.2.*</openmrsVersion>
 		</conditionalResource>
+
+		<conditionalResource>
+			<path>/lib/coreapps-api-pre2.2-${project.parent.version}.jar</path>
+			<openmrsVersion>2.2.*</openmrsVersion>
```

Comment:
This should be for versions from 1.9.* to 2.1.*

Reply:
@dkayiwa i have made changes to the versions.